impeller/renderer/backend/vulkan/capabilities_vk.cc

```diff
@@ -41,7 +41,7 @@ CapabilitiesVK::CapabilitiesVK(bool enable_validations) {
   validations_enabled_ =
       enable_validations && HasLayer("VK_LAYER_KHRONOS_validation");
   if (enable_validations && !validations_enabled_) {
-    FML_LOG(ERROR)
+    FML_LOG(FATAL)
```

Comment:
I think the safer thing is to add a test that makes sure this doesn't get printed out.  I'm concerned about sending users down this path on accident.

Reply:
For now I'm keeping this fatal for testing.

So we have two approaches here:

1. Keep the existing behavior but add checks that it is running as expected on CI.
2. Make it fatal but do more work to make sure users don't fall down the path.

Reply:
What I'm doing for impeller validations is that it is off by default (existing behavior) but I swap it on in the `main()` for the test runner and adding a test to make sure it it is on.

Reply:
I'm moving this back to error, as a follow up we need to make sure accidentally turning off validations causes a test somewhere to fail - ideally once for each shard.

Reply:
https://github.com/flutter/flutter/issues/145041